invenio_app_ils/schemas/documents/document-v1.0.0.json

```diff
@@ -82,6 +82,215 @@
           "type": "string"
         }
       }
+    },
+    "document_types": {
+      "items": {
+        "type": "string",
+        "enum": ["BOOK", "PROCEEDINGS", "STANDARD"],
+        "default": "BOOK"
+      },
+      "minItems": 1,
+      "title": "Document types",
+      "type": "array",
+      "uniqueItems": true
+    },
+    "language": {
```

Comment:
This is singular or plural https://github.com/CERNDocumentServer/cds-dojson/blob/97abcdcffed04e695d69d15f0c42117d255cc407/cds_dojson/marc21/fields/books/base.py#L480 ? Sorry maybe I am completely lost with the migration rules :P